corehq/apps/domain/management/commands/calculate_physical_size.py

```diff
@@ -13,11 +13,13 @@
 
 
 class Command(BaseCommand):
-    help = "Estimates the physical size of a domain's forms and cases."
+    help = "Estimates the physical size of one or more domains' forms and cases."
 
     def add_arguments(self, parser):
         parser.add_argument(
-            'domain',
+            'domains',
+            help="Enter a list of domains surrounded by quotation marks with only spaces between the domains."
+            "A single domain can also be entered",
```

Comment:
Nit: match the phrasing of the other arguments. For example, `sample_size` is: `the number of samples to average over per form/case type`

I think `a white-space-separated list of domains` works.

Non-nit: I didn't realize we were still using a single parameter for reading an array. [argparse](https://docs.python.org/3/library/argparse.html#the-add-argument-method) (the parser Django uses under-the-hood) exposes `nargs` for this. adding `nargs='+' to `add_argument` should eliminate the need for the split, and make the command interface more intuitive to the user.